Libraries/LibCore/StandardPaths.cpp

```diff
@@ -237,49 +237,4 @@ ByteString StandardPaths::tempfile_directory()
 #endif
 }
 
-ErrorOr<Vector<String>> StandardPaths::font_directories()
```

Comment:
this commit should be able to remove FontDirectory.h for Haiku from StandardPaths.cpp